src/modules/utils/string.ts

```diff
@@ -0,0 +1,2 @@
+export const showFirstAndLastFour = (str: string, isLowerThanEight = str.length <= 8) =>
```

Comment:
Is this used for anything else than truncating pub keys? Maybe we should rename it to truncatePubkey or shortenPubkey?